### PR TITLE
solution

### PR DESCRIPTION
**Pull request by tiserett**
[DEMO LINK](https://tiserett.github.io/react_decompose/)

### CODE REVIEW

**Review by yungdena — APPROVED** [review]
Nice one🔥

File: src/App.css [hunk 1]
```diff
@@ -14,74 +14,6 @@ body {
...
-  margin: 0;
-  color: #333;
-}
-
-.article__paragraph:not(:first-child) {
-  margin-top: 0.7em;
-}
-
 @media (min-width: 500px) {
   .header {
```
Comment (AlinaMatuschak) [inline src/App.css:18]: it should move to the header file

Comment (AlinaMatuschak) [reply src/App.css:18]: and next selector to article file

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Almost perfect )

**Pull request closed without merging**